docker/de.bsi.secvisogram.json

```diff
@@ -0,0 +1,7 @@
+{
+  "loginAvailable": false,
+  "loginUrl": "",
+  "logoutUrl": "",
+  "userInfoUrl": "",
+  "validatorUrl": "http://localhost:8082"
```

Comment:
I'm not sure if this is flexible enough. It might be better to provide a default without validator here and add documentation for the user provide their own via a "volume".